frontend/public/help/en/instance.md

```diff
@@ -0,0 +1,7 @@
+---
+title: How to setup 'Instance' ?
+---
+
+- Each Bytebase instance belongs to an environment. An instance usually maps to one of your database instance represented by an host:port address. This could be your on-premises MySQL instance, a RDS instance.
```

Comment:
```suggestion
- Each Bytebase instance belongs to an environment. An instance usually maps to one of your database instance represented by an host:port address. This could be your on-premises MySQL instance or RDS instance.
```